notebooks/SS_image_scraper_exploratory_notebook_RJProctor.ipynb

```diff
@@ -0,0 +1,398 @@
+        "# print out HTML\r\n",
+        "contents = webpage.prettify()\r\n",
+        "\r\n",
+        "print(contents)\r\n"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "JqJZAsXEOaqN"
+      },
+      "source": [
+        "grab an image using url...grab grade tags for grade 3, grade 4, grade 5"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "ZwNy1k3qaa_R"
+      },
+      "source": [
+        "grabbing grade level tags:\r\n",
+        "\r\n",
+        "--found in div class grid item\r\n",
+        "\r\n",
+        "\r\n",
+        "*   table\r\n",
+        "*   tbody\r\n",
+        "*   tr\r\n",
+        "*   td\r\n",
+        "\r\n",
+        "use a.get_text() to grab\r\n",
+        "\r\n",
+        "Data cleanning --> \r\n",
+        "replace (['(',')'], [' ', ' '])\r\n",
+        "and \r\n",
+        "(' ', strip=True)\r\n",
+        "\r\n",
+        "a.get_text(' ', strip=True).replace (['(',')'], [' ', ' '])\r\n",
+        "\r\n"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "epNer9k8P7IL"
+      },
+      "source": [
+        "# select information contained in the page table\r\n",
+        "table = webpage.findAll(lambda tag: tag.name=='table') \r\n",
+        "\r\n",
+        "\r\n",
+        "# get a list of table rows from page table\r\n",
+        "#rows = table.find(lambda tag: tag.name=='tr')\r\n",
+        "\r\n",
+        "# rows = []\r\n",
+        "# for \r\n",
+        "\r\n",
+        "\r\n",
+        "#images = webpage.findAll(name='img')\r\n",
+        "\r\n",
+        "#images = webpage.find(name=\"img\", attrs={'class':'grid-item-art'})\r\n",
+        "\r\n",
+        "# select all elements with an a attribute inside div class grid item\r\n",
+        "#artworks = [img for img in images if img['src']]\r\n",
+        "#artworks = [img['src'] for img in images]\r\n",
+        "\r\n",
+        "\r\n",
+        "# grab text label for grid item\r\n",
+        "#grid_item_label = grid_item_info.get_text()\r\n",
+        "\r\n",
+        "#print(grid_item_info)\r\n",
+        "print(len(rows))\r\n",
+        "print(table)"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "Qmqw8B_tg9L5"
+      },
+      "source": [
+        "1. may need to grab grade level first to create dictionary \r\n",
+        "2. work on grabbing a single image\r\n",
+        "3. create function/loop to work with label grab"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "TumSwomNrZr4"
+      },
+      "source": [
+        "There are images hidden inside a table that can be seen, but accessed using Beautiful Soup alone.  To overcome this obstacle, I will use a python package that automates a web browser, Selenium.\r\n"
```

Comment:
I can't see the rest of this line; could you insert line break(s)?

Reply:
This is markdown and auto-formats to the size of the screen when viewing in the notebook.  Unfortunately, when reviewing in GitHub, we have to scroll.